packages/calcite-components/src/components/dropdown/dropdown.tsx

```diff
@@ -675,4 +672,10 @@ export class Dropdown
       this.el.addEventListener("calciteDropdownOpen", this.toggleOpenEnd);
     }
   };
+
+  private updateTabIndexOfItems(event: CustomEvent<ItemKeyboardEvent>): void {
```

Comment:
To make it more general and decouple the event from this, can you update the signature to pass the item directly instead of the event?

Reply:
makes sense. updated.